config/platforms-chromeos.yaml

```diff
@@ -8,7 +8,7 @@ _anchors:
         url: https://storage.chromeos.kernelci.org/images/kernel/v6.1-{mach}
         image: 'kernel/Image'
       nfsroot: https://storage.chromeos.kernelci.org/images/rootfs/debian/bookworm-cros-flash/20240422.0/{debarch}
-      tast_tarball: https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-{base_name}/20240514.0/{debarch}/tast.tgz
+      tast_tarball: https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-{base_name}/202409??.0/{debarch}/tast.tgz
```

Comment:
@denisyuji @nuclearcat this PR is just a placeholder and reminder that we need to update the URLs once all work is complate. Feel free to close this one and open yours in case you want to merge this quickly and then.

Reply:
new directory is 20240918.0, can you update it please?

Reply:
Updated the PR.